shuup/front/templates/shuup/front/product/category.jinja

```diff
@@ -5,23 +5,28 @@
 {% else %}
     {% extends "shuup/front/base.jinja" %}
     {% block extrameta %}
-    {{ macros.render_metadata(category, category.name, category.description, "category") }}
+    {% if category %}
+        {{ macros.render_metadata(category, category.name, category.description, "category") }}
+    {% endif %}
     {% endblock %}
-    {% block title %}{{ category.name }}{% endblock %}
+    {% block title %}{{ (category.name if category else _("Categories")) }}{% endblock %}
 
-    {% block content_title %}{{ category.name }}{% endblock %}
+    {% block content_title %}{{ (category.name if category else "") }}{% endblock %}
 
     {% block breadcrumb %}
-        {% call macros.breadcrumb() %}
-            {{ macros.render_breadcrumb_item("/", _("Home")) }}
-            {% for c in category.get_ancestors() if c.is_visible(customer=request.customer) %}
-                {{ macros.render_breadcrumb_item(shuup.urls.model_url(c), c.name) }}
-            {% endfor %}
-            {{ macros.render_breadcrumb_item(None, category.name, True) }}
-        {% endcall %}
+        {% if category %}
+            {% call macros.breadcrumb() %}
+                {{ macros.render_breadcrumb_item("/", _("Home")) }}
```

Comment:
IMO we should keep at least breadcrumbs to Home / Categories

Reply:
It is better this way IMO. This was also my first idea, but it did look awkward